app/src/main/java/at/tacticaldevc/oat/utils/Prefs.java

```diff
@@ -27,16 +34,125 @@
     private final static String DOCUMENT_NAME_ENABLED_FEATURES = "oat-enabled-features";
     private final static String DOCUMENT_NAME_ACCEPTED_CONDITIONS = "oat-accepted-conditions";
     private final static String KEY_TRUSTED_CONTACTS = "trusted-contacts";
+    private final static String KEY_COMMAND_PASSWORD = "password";
+    private final static String KEY_COMMAND_PASSWORD_SALT = "pwdsalt";
+    private final static String KEY_COMMAND_TRIGGER = "cmd-trigger";
+    private final static String KEY_MISSING_PERMISSIONS_TO_REQUEST_ON_STARTUP = "missing-permission";
+
+    // Basic Data
+
+    /**
+     * Updates the Password used to send commands to the App
+     *
+     * @param context  the Context of the Application
+     * @param password the new Password
+     */
+    public static void savePassword(Context context, String password) {
```

Comment:
Idk maybe it would make more sense to ask and verify the old password. Well... Just a thought